File: api/examples/c#/BUBBLE_QUERY.cs
using System;
using System.Collections.Generic;

[Serializable()]
public partial class BUBBLE_QUERY
{
    public long BrowserID = 0L;
    public int BrowserPID = 0;
    public long DNSID = 0L;
    public List<long> ProxyList = new List<long>();
    public long VPNID = 0L;
    public string ProfilePath = string.Empty;
    public LAUNCH_RESULT PreLaunch = new LAUNCH_RESULT();
    public LAUNCH_RESULT PostLaunch = new LAUNCH_RESULT();
    public int Options = 0;
    public int BorderColor = 0;
    public Dictionary<int, string> SpoofSettings = new Dictionary<int, string>();
}

File: api/examples/vb.net/BUBBLE_QUERY.vb
<Serializable()>
Public Class BUBBLE_QUERY
    Public InstanceID As Long = 0
    Public Name As String = String.Empty
    Public BrowserID As Long = 0
    Public BrowserPID As Integer = 0
    Public DNSID As Long = 0
    Public ProxyList As New List(Of Long)
    Public VPNID As Long = 0
    Public ProfilePath As String = String.Empty
    Public PreLaunch As New LAUNCH_RESULT
    Public PostLaunch As New LAUNCH_RESULT
    Public Options As Integer = 0
    Public BorderColor As Integer = 0
    Public SpoofSettings As New Dictionary(Of Integer, String)
End Class
